cli/pawls/commands/metric.py

```diff
@@ -343,10 +367,13 @@ def metric(
             coco_results, coco_category_results = coco_eval.calculate_ap_scores()
 
             if verbose:
-                coco_eval.show_results(coco_results)
+                save_table = coco_eval.show_results(coco_results)
```

Comment:
This method name could be changed if it no longer shows the results

Reply:
Yes, that's a great point!

Reply:
I double checked - This function still prints the table, and also returns the table. I feel like something like `print_and_savet_results` should be better - but just a bit awkward? We could come back and fix this issue later after we find a better strategy for saving the reports.